names/images.go

```diff
@@ -0,0 +1,16 @@
+package names
+
+import (
+	"fmt"
+
+	"github.com/docker/distribution/reference"
+)
+
+func NormalizeImageName(image string) (string, error) {
+	named, err := reference.ParseNormalizedNamed(image)
```

Comment:
Why do we need a function to wrap the object?
 Consider calling the [Parse](https://github.com/distribution/distribution/blob/main/reference/reference.go#L191) function instead. This way we will be able to extract the [tag](https://github.com/distribution/distribution/blob/main/reference/reference.go#L370), [digest](https://github.com/distribution/distribution/blob/main/reference/reference.go#L374) and [name](https://github.com/distribution/distribution/blob/main/reference/reference.go#L366).
This leads me to think, why are we writing wrapper and not calling directly the `reference` package

Reply:
the goal here is to add `docker.io/library` every time we read an image from a k8s object... to make sure we have a repo
`Parse` can be used in kubevuln to parse an image, but it doesn't look like it will add the default repo if needed... (checking quickly)

Reply:
no it doesn't: https://go.dev/play/p/quZiEbVAO_2